pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -864,7 +864,12 @@ func TestGarbageCollectorSync(t *testing.T) {
 	//            gc.resyncMonitors()
 	//            controller.WaitForCacheSync() loops with `syncedPollPeriod` (hardcoded to 100ms), until either its stop channel is closed after `period`, or all caches synced.
 	//
-	// Setting the period to 200ms allows the WaitForCacheSync() to check for cache sync ~2 times in every wait.PollImmediateUntil() loop.
+	// Setting the period to 200ms allows the WaitForCacheSync() to check
+	// for cache sync ~2 times in every wait.PollImmediateUntil() loop.
```

Comment:
Is this at least 2?

Reply:
Unfortunately, this is really a "roughly 2" due to the nature of golang `select`.